frontend/src/styles/Login.css

```diff
@@ -132,27 +136,80 @@ body {
   justify-content: center;
 }
 
-.or {
+/* .or {
   margin-top: 0;
   width: 100%;
-  font-size: 30px;
+  font-size: 500px;
   text-align: center;
   color: white;
   font-weight: bold;
-}
+} */
 
-.oauthButton {
+/* .oauthButton {
   width: 60%;
   padding: 1.3em;
   margin-bottom: 1em;
   display: flex;
   justify-content: space-between;
-}
+} */
```

Comment:
Remove comment if it's unnecessary. If you have a reason why you want to keep it, add a comment before explaining why this is commented out.